app/Http/Requests/UpdateIndividualConstituenciesRequest.php

```diff
@@ -17,114 +19,145 @@ public function authorize(): bool
+            'disability_and_deaf.required_without' => __('You must select at least one option for “Can you connect to people with disabilities, Deaf persons, and/or their supporters?”'),
+            'lived_experience_connections.required_if' => __('You must select at least one option for “Can you connect to people with disabilities, Deaf persons, and/or their supporters?”'),
+            'base_disability_type.required_if' => __('You must select one option for “Please select people with disabilities that you can connect to”.'),
+            'disability_and_deaf_connections.required' => __('You must select which people with specific disabilities and/or Deaf people you can connect to.'),
+            'area_type_connections.required' => __('You must select at least one option for “Where do the people that you can connect to come from?”'),
+            'has_indigenous_connections.required' => __('You must select one option for “Can you connect to people who are First Nations, Inuit, or Métis?”'),
+            'indigenous_connections.required_if' => __('You must select at least one Indigenous group you can connect to.'),
             'refugees_and_immigrants' => __('You must select one option for “Can you connect to refugees and/or immigrants?”'),
-            'has_gender_and_sexual_identities.required' => __('You must select one option for “Can you connect to people who are marginalized based on gender or sexual identity?”'),
-            'gender_and_sexual_identities.required_if' => __('You must select at least one gender or sexual identity group you can connect to.'),
-            'has_age_brackets.required' => __('You must select one option for “Can you connect to a specific age group or groups?”'),
-            'age_brackets.required_if' => __('You must select at least one age group you can connect to.'),
-            'has_ethnoracial_identities.required' => __('You must select one option for “Can you connect to people with a specific ethno-racial identity or identities?”'),
-            'ethnoracial_identities.required_if' => __('You must select at least one ethno-racial identity you can connect to.'),
-            'constituent_languages.*.in' => __('You must select a language.'),
+            'has_gender_and_sexuality_connections.required' => __('You must select one option for “Can you connect to people who are marginalized based on gender or sexual identity?”'),
+            'gender_and_sexuality_connections.required' => __('You must select at least one gender or sexual identity group you can connect to.'),
+            'nb_gnc_fluid_identity.required' => __('You must select at least one gender or sexual identity group you can connect to.'),
+            'has_age_bracket_connections.required' => __('You must select one option for “Can you connect to a specific age group or groups?”'),
+            'age_bracket_connections.required' => __('You must select at least one age group you can connect to.'),
```

Comment:
I think one of these isn't working or maybe there is a message missing as the error message returned when you don't specify an age group is "The age bracket connections field is required when has age bracket connections is 1."

<img width="932" alt="incorrect validation error message" src="https://user-images.githubusercontent.com/556024/206480972-a6730081-fa9c-483e-8fd5-2f27051469b5.png">